custom_components/mixergy/sensor.py

```diff
@@ -423,6 +424,23 @@ def unit_of_measurement(self):
     def name(self):
         return f"Mixergy Electric PV Power"
 
+class PVEnergySensor(IntegrationSensor):
+
+    def __init__(self, tank:Tank):
+        super().__init__(
+            name="Mixergy Electric PV Energy",
+            source_entity="sensor.mixergy_electric_pv_power",
+            round_digits=2,
+            unit_prefix=None, # PVPowerSensor is already in kW
```

Comment:
It's a little annoying that the PV power sensor is in kW and not W, as it's then different to the main power sensor. But I figured that it is best to not change that as it would be a breaking change for people.